src/main/java/org/jenkinsci/plugins/configfiles/buildwrapper/ManagedFileUtil.java

```diff
@@ -97,12 +105,12 @@ public FilePath call() throws IOException {
 
             FilePath target = null;
             if (createTempFile) {
-                target = ManagedFileUtil.createTempFile(build.getWorkspace().getChannel());
+                target = ManagedFileUtil.createTempFile(ws.getChannel());
             } else {
                 
                 String expandedTargetLocation = managedFile.targetLocation;
                 try {
-                    expandedTargetLocation = TokenMacro.expandAll(build, listener, managedFile.targetLocation);
+                    expandedTargetLocation = TokenMacro.expandAll((AbstractBuild<?, ?>) build, listener, managedFile.targetLocation);
```

Comment:
Thx for response.
At the moment I use the plugin without the TokenMacro calls... just for testing...it seems to work on the slave nodes... 

I'm not really sure about  the default approach, should I wait for a api fix or contact the developer?